src/lib/pages/frontsite/Documents/DocumentDetails/DocumentDetails.js

```diff
@@ -186,7 +186,11 @@ class DocumentDetails extends Component {
         >
           <Container fluid className="literature-search-container">
             <Container>
-              <SearchBarILS onSearchHandler={this.onSearchClick} />
+              <SearchBarILS
+                onSearchHandler={this.onSearchClick}
+                placeholder="Search for books, series, articles, publications..."
```

Comment:
👍  I see you repeat it in many places, maybe it should be a default prop of SearchBarILS?

Reply:
`SearchBarILS` is used in different places (scanning for instance), reason why I opted for this quick fix.